tests/test_prediction.py

```diff
@@ -70,6 +74,16 @@ def test_repr(test_object: any):
         for _ in TEST_DEFAULT_CATEGORY_PREDICTIONS
     ]
 
+    [
+        test_repr(SceneCategoryPrediction.from_json(_))
+        for _ in TEST_SCENE_CATEGORY_PREDICTIONS
+    ]
+
+    [
+        test_repr(SceneCategoryPrediction.from_json(_))
+        for _ in TEST_DEFAULT_SCENE_CATEGORY_PREDICTIONS
+    ]
+
```

Comment:
I'd not add more `test_repr`, there is no reason for the `repr` looking like something else in the IDE. I've been slowly removing these as they break, it is often useful to add extra info to the `repr` and making it more rigid to change serves no one.